Modules/timemodule.c

```diff
@@ -414,6 +414,8 @@ Return the clk_id of a thread's CPU time clock.");
 static PyObject *
 time_sleep(PyObject *self, PyObject *timeout_obj)
 {
+    PySys_Audit("time.sleep", "O", timeout_obj);
```

Comment:
Why no error handling?

Reply:
I could tell you but it would sound like a lame excuse.  (I picked up a low-priority task from May, and didn't realize how much work-in-progress it was...)

Thank you very much for spotting this! Fix incoming.

Reply:
Fix in #108363.